### DISCUSSION TITLE
XCA 2.6.0 Seems Very Slow - Running on Windows 10

### DISCUSSION THREAD

**Discussion by GiladHi** [chris2511/xca#540] [Q&A] [answered]
Hi,

Just installed the latest build of XCA, version 2.6.0, to replace the previous installation of version 2.5.0.

It seems like every option on the GUI takes a lot of resources, for some reason...
I have around 300 certificates. For example, when trying to sort by expiry date it takes between 6-10 seconds, and the interface freezes.

With version 2.5.0 is was instantly.

Any ideas how to solve?

Thanks!

(other than that - amazing application!)

**Comment by chris2511** [comment, marked as answer]
Please see #535. There is also a fix/workaround.

**Comment by GiladHi** [comment]
Wow, this must be the fastest reply to a support ticket ever! :)

I just installed the QT 5.15 build and everything seems ok.

Thanks a lot!

**Comment by GiladHi** [comment]
Hi Chris, 

Any chance for a XCA build based on the latest QT version (6.7.1)? I still have performance issues on Windows (really need the colours to highlight the expired certificates)...

Thanks!
